docs/docs/Developing_with_JAC/Overview.md

```diff
@@ -9,26 +9,26 @@ Jaseci is a computational Model and language in one.Jaseci provides the ability
 
 ## Feature of Jaseci
 
-1. <strong>Easy to learn</strong> :
+1.<strong>Easy to learn: </strong> :
 Jaseci is a high level programming language. It is very easy to learn. The paradigm for Jaseci is based upon graphs.Once you underastand this new paradigm the language is simple.
```

Comment:
Jaseci uses JAC, an easy-to-learn "glue" language that's based on Python and Javascript. With JAC, and the level of abstraction it brings, you can wield advanced AI Engines and deliver API payloads with just a few lines of code!